src/components/AppSidebar/AppSidebar.vue

```diff
@@ -176,19 +176,23 @@ import Actions from '../Actions'
 import Focus from '../../directives/Focus'
 import Linkify from '../../directives/Linkify'
 import l10n from '../../mixins/l10n'
-import AppSidebarTabs from '../AppSidebarTabs/AppSidebarTabs'
+import AppSidebarTabs from './AppSidebarTabs'
```

Comment:
This won't work, since we don't export `AppSidebarTabs`. I understood your comment https://github.com/nextcloud/nextcloud-vue/pull/1281#discussion_r469755383 as not exporting `AppSidebarTabs`. Did you mean something else?

Reply:
Well, it does work. I moved it alongside the AppSidebar.vue file.
Yes, we misunderstood each other, I think it doesn't make sense to have it in a dedicated folder if we don't export it :)

Reply:
Ah ok, but then you need to adjust the tests as well: https://github.com/nextcloud/nextcloud-vue/blob/master/tests/unit/components/AppSidebar/AppSidebarTabs.spec.js#L26 fails at the moment.